src/main/java/game/Car.java

```diff
@@ -1,18 +1,30 @@
 package game;
 
 public class Car {
-    private final int id;
+    private final CarName carName;
     private int position;
     private final MovePolicy movePolicy;
 
-    public Car(int id, int position, MovePolicy movePolicy) {
-        this.id = id;
+    private static final int MOVING_RANGE = 1;
+
+    public Car(CarName carName, int position, MovePolicy movePolicy) {
+        this.carName = carName;
         this.position = position;
         this.movePolicy = movePolicy;
     }
 
     public void move() {
-        position = movePolicy.getPosition(position);
+        if (movePolicy.isMovable()) {
+            position += MOVING_RANGE;
+        }
+    }
+
+    public CarName getCarName() {
+        return carName;
+    }
+
+    public String getCarNameValue() {
+        return carName.getName();
     }
```

Comment:
현재 CarName에 대해 getter가 2개가 생기게 되었습니다.
개인적으로는 CarName은 carName 값을 가진 값 객체이므로 CarName 객체 자체에 대한 getter를 생성하는것보다
CarName의 name을 바로 반환해주는것도 좋을 것 같아요 :)

```suggestion
    public String getCarName() {
        return carName.getName();
    }
```

Reply:
넵 리뷰어님 이것과, 일급 컬렉션 사용에 대해서 5단계 PR에서 질문을 드린게 있는데, 나중에 확인 부탁드립니다 ~